pkg/cloudprovider/gce/gce.go

```diff
@@ -276,13 +276,18 @@ func (gce *GCECloud) targetPoolURL(name, region string) string {
 
 func waitForOp(op *compute.Operation, getOperation func() (*compute.Operation, error)) error {
 	pollOp := op
+	consecPollFails := 0
 	for pollOp.Status != "DONE" {
 		var err error
-		// TODO: add some backoff here.
```

Comment:
Did the need to back off go away for some reason? I know it's not your code, but polling every second seems like way more than we need, given that the things this function waits on take tens of seconds (if not minutes) to complete.